src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1389,18 +1389,14 @@
      * class object.
      *
      * <p>For example:</p>
-     *
-     * <pre>
-     * int.class
+     * <pre> int.class
      * </pre>
      * <p>parses as:</p>
-     * <pre>
-     * +--EXPR
-     *     |
-     *     +--DOT (.)
-     *         |
-     *         +--LITERAL_INT (int)
-     *         +--LITERAL_CLASS (class)
+     * <pre> 
+     * |`--EXPR -> EXPR
+     * |    `--DOT -> .
+     * |        |--LITERAL_INT -> int
+     * |        `--LITERAL_CLASS -> class
```

Comment:
Please remove `|` as it does not lead anywhere.

Reply:
commit 3fef3f2 has
1. Removed |
2. -> is replaced with -&gt; since Javadoc processes it as a closing tag